gravitee-apim-console-webui/docs/management-application-subscriptions.md

```diff
@@ -2,12 +2,12 @@
 
 Retrieve all the registered subscriptions per API. Subscription represents the plans the application has subscribed to.
 
-Subscriptions can be : 
+Subscriptions can be :
```

Comment:
```suggestion
Subscriptions can be:
```

Reply:
Nice catch! I missed some of these, will do a 2nd find and replace